packages/static-hosting/lib/static-hosting.ts

```diff
@@ -560,6 +560,31 @@ export class StaticHosting extends Construct {
       }
     }
 
+    if (enableStaticFileRemap) {
+      const staticFileRemapPrefixes = props.defaultBehaviourPrefixes?.map(
+        prefix => `${prefix.prefix}/`
+      ) || [""];
+      staticFileRemapPrefixes.forEach(prefix => {
+        this.staticFiles.forEach(path => {
+          additionalBehaviors[`${prefix}*.${path}`] = {
+            origin: s3Origin,
+            viewerProtocolPolicy: ViewerProtocolPolicy.REDIRECT_TO_HTTPS,
+          };
+        });
+      });
+    }
+
+    props.defaultBehaviourPrefixes?.forEach(prefix => {
```

Comment:
Pretty sure this won't work with a blank string :thinking: And if it did it wouldn't be the desired behaviour

Reply:
Never mind, I thought this was in the if statement :sweat_smile: If the user passes a blank string then that's on them haha